app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __str__(self) -> str:
+        return "Name: {name}, Health: {health}, Hidden: {hidden}".format(
+            name=self.name, health=self.health, hidden=self.hidden
+        )
+
+    def __repr__(self) -> str:
+        return "{{Name: {name}, Health: {health}, Hidden: {hidden}}}".format(
+            name=self.name, health=self.health, hidden=self.hidden
+        )
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        if self.hidden is True:
+            self.hidden = False
+        else:
+            self.hidden = True
```

Comment:
Please check the checklist before you send your work on review.

![image](https://user-images.githubusercontent.com/104218550/201746892-e46c3e0c-303a-443d-9fea-959e71ef8216.png)